pom.xml

```diff
@@ -213,6 +213,7 @@
             -Xmx1g
             --add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED
             --add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED
+            --add-exports=jdk.compiler/com.sun.tools.javac.jvm=ALL-UNNAMED
```

Comment:
If this PR is accepted, then the [documentation](https://errorprone.info/docs/installation) should also be updated to include this flag.